doc/api/Daemon.md

```diff
@@ -83,10 +83,10 @@ returns the set of constants in use.
   "rootdepth": [255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255],
 
   // Largest allowed ratio between the old difficulty and the new difficulty.
-  "maxadjustmentup": "5/2",
+  "maxtargetadjustmentup": "5/2",
 
   // Smallest allowed ratio between the old difficulty and the new difficulty.
-  "maxadjustmentdown": "2/5",
+  "maxtargetadjustmentdown": "2/5",
```

Comment:
this is fine, we don't need to document the deprecated fields, so we don't need to have the duplication here even though we need the duplication in the actual code itself.